grails-app/domain/org/pih/warehouse/order/OrderAdjustment.groovy

```diff
@@ -100,27 +99,28 @@ class OrderAdjustment implements Serializable, Comparable<OrderAdjustment> {
         }
     }
 
-    Boolean getIsInvoiced() {
-        return !postedPurchaseInvoiceItems.empty
+    Boolean getIsFullyInvoiced() {
+        return Math.abs(unitPriceOnPostedInvoices) >= Math.abs(totalAdjustments)
     }
```

Comment:
I removed the `isInvoiced` transient because it is not used anywhere anymore and if we will ever need this in the future I think we are going to use the derived status like we use in other domains.